pkg/cli-runtime/testing/interfaces.go

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2014 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package testing
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+	client "k8s.io/client-go/rest"
+)
+
+// RESTClient is a client helper for dealing with RESTful resources
+// in a generic way.
+type RESTClient interface {
```

Comment:
why do you need this interface for testing? I dont see any corresponding fakes.

Reply:
We do not need these interfaces now as we are building our own fake builder. I have removed these